recipe/build.sh

```diff
@@ -60,9 +65,17 @@ done
 
 
 if [ "${mpi}" = 'nompi' ] ; then
-    gmx='gmx'
+    if [ "${double}" = 'no' ] ; then
+        gmx='gmx'
+    else
+        gmx='gmx_d'
+    fi
 else
-    gmx='gmx_mpi'
+    if [ "${double}" = 'no' ] ; then
+        gmx='gmx_mpi'
+    else
+        gmx='gmx_mpi_d'
```

Comment:
I think that it would be better to leave the binary as either `gmx` or `gmx_mpi` - I believe that these are standard names for gromacs, and `gmx_d` is not.

However I could be wrong in this - @mabraham, can you advise further on this?

Reply:
`gmx`, `gmx_d`, `gmx_mpi`, and `gmx_mpi_d` are the standard names. See https://gitlab.com/gromacs/gromacs/-/blob/master/cmake/gmxManageSuffixes.cmake

Reply:
Ahh, I'd not seen that - thanks.